templates/ampache.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>ampache</Name>
+  <Repository>ampache/ampache</Repository>
+  <Registry>https://hub.docker.com/r/ampache/ampache/</Registry>
+  <Network>bridge</Network>
+  <Shell>sh</Shell>
+  <Privileged>false</Privileged>
+  <Support>https://github.com/ampache/ampache/issues</Support>
+  <Project>https://ampache.org/</Project>
+  <Overview>Ampache ...for the love of music&#xD;
+  &#xD;
+  You will need to follow the configuration steps here once the container is running: https://github.com/ampache/ampache/wiki/Basic &#xD;
+  &#xD;</Overview>
+  <WebUI>http://[IP]:[PORT:80]</WebUI>
```

Comment:
```suggestion
  <WebUI>http://[IP]:[PORT:8084]</WebUI>
```

Port should always be the same as the `Default` value for the "Web UI" config entry.